src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -25,15 +25,17 @@ the main body of the message as well as a quote.
 </docs>
 
 <template>
-	<div
+	<li
```

Comment:
Please make sure to address the server's css that targets unordered lists

Reply:
here I didn't see any visual differences so we might be fine

another alternative would be to use "role=listitem" here, this way the styles would not change

Reply:
No no, that's fine :)